stages/org.osbuild.selinux

```diff
@@ -5,9 +5,14 @@ import sys
 
 import osbuild.api
 from osbuild.util import selinux
+from osbuild.util import parsing
 
+def main(args, options):
+    tree = args["tree"]
+    alt_root = options.get("alt_root")
```

Comment:
Fwiw, I'm exploring `org.osbuild.ostree.selinux` right now for this, it might be a better fit but not sure yet.